extensions/amp-date-display/1.0/test/test-amp-date-display.js

```diff
@@ -132,3 +163,50 @@ describes.realWin(
     });
   }
 );
+
+describes.sandboxed('amp-date-display 1.0: parseDateAttrs', {}, () => {
+  const DATE = new Date(1514793600000);
+  const DATE_STRING = DATE.toISOString();
+
+  let element;
+
+  beforeEach(() => {
+    element = document.createElement('amp-date-display');
+  });
+
+  it('should throw when no date is specified', () => {
+    expect(() => parseDateAttrs(element)).to.throw(/Invalid date/);
+  });
+
+  it('should throw when invalid date is specified', () => {
+    element.setAttribute('datetime', 'invalid');
```

Comment:
Let's add invalid cases for the other attributes too. As well as check for the value of [`now`](https://amp.dev/documentation/components/amp-date-display/#datetime).